src/routes/Sponsors.js

```diff
@@ -12,7 +12,7 @@ function ImgSpon({sponsors, sAmount})
 	let Sp = sponsors.images;
 	let allImg = Sp.map(SponsorImages => {
 		return(
-			<div>
+			<div className='sponsor-item'>
```

Comment:
This div needs a key attribute. This is a react requirement when you destructor an array in a return statement. Other than that everything looks good.